### ISSUE TITLE
Pretrained ICR KB (Knowledge Base)

### ISSUE THREAD

**Issue by alex-pl** [ACSPRI/queXF#3]
Hi there! We're using LimeSurvey for our surveys in studies. One of the surveys is printed out, utilizing the queXF export functionality of LimeSurvey. I've now set up queXF (using the docker image) to ease the process of digitizing the survey results and get them back into the LimeSurvey instance, having a common way and place where all the collected data is stored. Setting up everything took me a while but it works pretty well now. So first of all, let me say thank you for this very helpful project!

Because of limited resources I'm always thinking about how we can automate things. One more thing would be using the ICR functionality of queXF. Our problem is, that creating our own model would be way more time consuming than it might save us time.

Is there any pretrained ICR KB (Intelligent Character Recognition Knowledge Base) publically available for use?

**Comment by adamzammit** [comment]
Hi @alex-pl ,

Glad to hear you have the systems working.

queXF has some models available in the source, see: https://github.com/ACSPRI/queXF/tree/master/doc/icrdatabase

The ICR feature can be useful but even if you can get accuracy reasonably high, I have found it to be more trouble than it is worth from a time-consumption standpoint. We found that even when it was reasonably accurate, it make the work of people verifying the forms more difficult as it was harder for them to spot the errors, and they ended up just re-typing most things anyway.

Good luck though and hope those pre-trained databases are of use.

**Comment by alex-pl** [comment]
Hi @adamzammit,

thanks for the quick response! I was into other things in the meantime but I'm back on it now.

We'll do it manually in this case then. When we have text input at all, users often do not really use the predefined fields or we have multi line textboxes. Together with your advice it seems to be the better approach.

Maybe I'll have another look into the pre-trained models another time.
